pages/linux/udevadm.md

```diff
@@ -0,0 +1,28 @@
+# udevadm
+
+> Linux `udev` management tool.
+> More information: <https://www.freedesktop.org/software/systemd/man/udevadm>.
+
+- Monitor all device events:
+
+`sudo udevadm monitor`
+
+- Print `uevents` sent out by the kernel:
+
+`sudo udevadm monitor --kernel`
+
+- Print device events after being processed by `udev`:
+
+`sudo udevadm monitor --udev`
+
+- List attributes of a device:
+
+`sudo udevadm info --attribute-walk --path {{/dev/sda1}}`
+
+- Reload all `udev` rules:
+
+`sudo udevadm control --reload-rules`
+
+- Trigger all `udev` rules to run:
```

Comment:
Hey there! Welcome to tldr-pages, @aos.

What precisely does this do?

Reply:
Howdy ! This "re-applies" all `udev` rules to run. So here's the general workflow:

1. You write a new rule and add it to `/etc/udev/rules.d/`
2. You reload all rules using `udevadm control --reload-rules` so that `udev` stores it in its in-memory database (this only usually happens on boot)
3. You tell `udev` to re-apply all the rules in its database, or "trigger" them to run

Reply:
Ah, ok. Thanks for the explanation! This seems ok to me then.